src/main/scala/net/cakesolutions/CakeBuildPlugin.scala

```diff
@@ -85,13 +83,15 @@ object CakeBuildPlugin extends AutoPlugin {
       scalaOrganization.value % "scala-reflect" % scalaVersion.value,
       scalaOrganization.value % "scalap" % scalaVersion.value
     ),
+    packageOptions in (Compile, packageBin) +=
+      Package.ManifestAttributes(CakeBuildInfoKeys.generalInfo.toSeq: _*),
```

Comment:
Yes. Buildinfo provides data available at runtime. The manifest attributes are present to enable production debugging issues - e.g. by allowing build lineage to be determined without the need for inferring the same data.